AdyenUI/Styles/AdyenSwitchStyle.swift

```diff
@@ -7,7 +7,7 @@
 import UIKit
 
 /// Contains the styling customization options for a switch item in a form.
-package struct AdyenToggleStyle {
+public struct AdyenSwitchStyle {
```

Comment:
Why is this public while the initializer stays at a package level?
If this is gonna be used by merchants, should be initializer be public too?

Reply:
Good point, I now see all styles are not aligned in terms of internal access modifiers, like label has everything public and buttons have everything package. Let me revisit it.